### PR TITLE
Mri header info additional fields

### PR DESCRIPTION
**Pull request by rathisekaran**

### CODE REVIEW

**Comment by Jkat** [comment]
Needs to be rebased to include the updated generic_includes.php

**Comment by rathisekaran** [comment]
Justin - i rebased this pull request.

File: php/libraries/FeedbackMRI.class.inc [hunk 1]
```diff
@@ -202,7 +201,7 @@ class FeedbackMRI
         $DB =& Database::singleton();
 
         // build the query
-        $query = "SELECT PredefinedCommentID, Comment, FieldName
+        $query = "SELECT PredefinedCommentID, Comment
```
Comment (driusan) [inline php/libraries/FeedbackMRI.class.inc]: Why is this removed?

File: tools/CouchDB_Import_MRI.php [hunk 2]
```diff
@@ -65,59 +65,100 @@ function updateDataDict($types)
...
             foreach ($feedback_types as $CommentTypeID=>$comments) {
                 if (!empty($comments['field'])) {
                     $fieldName = $comments['field'];
                     $type      = "enum ('" .implode("','", $comments['values'])."')";
 
-                    $this->Dictionary["Comment_".$fieldName."_$ScanType"] = array(
-                        'Type' => 'varchar(255)',
-                        'Description' => "Overall Comment for $fieldName $ScanType"
-                        );
+                    $cmt_field = "Comment_".$fieldName."_$ScanType";
```
Comment (driusan) [inline tools/CouchDB_Import_MRI.php]: We should probably make this script consistent with how it names things. Starting with $ScanType_$FieldName would make sorting easier. (All the scantype fields would show up together in the DQT.)

File: tools/CouchDB_Import_MRI.php [hunk 3]
```diff
@@ -65,59 +65,100 @@ function updateDataDict($types)
...
+                    $this->Dictionary[$preDef_field] = $preDef_Array;
+                    $pre[$preDefinedCommentTypeID]   = $preDefinedComment['field'];
                 }
                 $this->feedback_PreDefinedComments[$CommentTypeID] = $pre;
 
             }
+            $mri_array = array(
+                          'ScannerID'           => 'Scanner ID',
+                          'Pipeline'            => 'Pipeline',
+                          'OutputType'          => 'Output',
```
Comment (driusan) [inline tools/CouchDB_Import_MRI.php]: Missing "Type" in the description portion

File: tools/CouchDB_Import_MRI.php [hunk 4]
```diff
@@ -65,59 +65,100 @@ function updateDataDict($types)
...
+                          'AcquisitionDate'     => 'Acquisition Date',
+                          'FileInsertDate'      => 'Insert date',
+                          'SeriesDescription'   => 'Series Description',
+                          'SeriesNumber'        => 'Series Number',
+                          'EchoTime'            => 'Echo Time',
+                          'RepetitionTime'      => 'Repetition Time',
+                          'SliceThickness'      => 'Slice Thickness',
+                          'Time'                => 'No of volumes',
+                          'Comment'             => 'Comment',
+                          'SlicewiseRejected'   => 'Slicewise correlations (Nb)',
```
Comment (driusan) [inline tools/CouchDB_Import_MRI.php]: What does (Nb) mean?

File: tools/CouchDB_Import_MRI.php [hunk 5]
```diff
@@ -65,59 +65,100 @@ function updateDataDict($types)
...
+                          'InterlaceRejected'   => 'Interlace correlations (Nb)',
+                          'IntergradientReject' => 'Gradient-wise correlations (Nb)',
+                          'TotalRejected'       => 'No. of rejected directions',
+                          'Caveat'              => 'Caveat',
+                          'ProcessingPipeline'  => 'Processing Pipleline',
+                         );
+            $this->mri_header_fields = $mri_array;
+            foreach ($this->mri_header_fields as $field=>$desc) {
+                $mri_field = $field."_$ScanType";
```
Comment (driusan) [inline tools/CouchDB_Import_MRI.php]: I think these should be $ScanType_$field, not $field_$ScanType

File: php/libraries/FeedbackMRI.class.inc (hunk 1 above)
Comment (rathisekaran) [inline php/libraries/FeedbackMRI.class.inc]: FieldName column does not exist in the table, this was throwing an exception when select was executed
